Analysis/Tasks/PWGHF/qaTaskEfficiency.cxx

```diff
@@ -65,84 +61,74 @@ void makelogaxis(T h)
 
 /// Task to QA the efficiency of a particular particle defined by particlePDG
 template <o2::track::pid_constants::ID particle>
-struct QATrackingEfficiencyPt {
+struct qaTrackingEfficiency {
   static constexpr PDG_t PDGs[5] = {kElectron, kMuonMinus, kPiPlus, kKPlus, kProton};
   static_assert(particle < 5 && "Maximum of particles reached");
   static constexpr int particlePDG = PDGs[particle];
-  o2fw::Configurable<float> etaMin{"eta-min", -3.f, "Lower limit in eta"};
-  o2fw::Configurable<float> etaMax{"eta-max", 3.f, "Upper limit in eta"};
-  o2fw::Configurable<float> phiMin{"phi-min", 0.f, "Lower limit in phi"};
-  o2fw::Configurable<float> phiMax{"phi-max", 2.f * M_PI, "Upper limit in phi"};
-  o2fw::Configurable<float> ptMin{"pt-min", 0.f, "Lower limit in pT"};
-  o2fw::Configurable<float> ptMax{"pt-max", 5.f, "Upper limit in pT"};
-  o2fw::Configurable<int> ptBins{"pt-bins", 500, "Number of pT bins"};
-  o2fw::Configurable<int> logPt{"log-pt", 0, "Flag to use a logarithmic pT axis"};
-  o2fw::Configurable<int> etaBins{"eta-bins", 500, "Number of eta bins"};
-  o2fw::Configurable<int> phiBins{"phi-bins", 500, "Number of phi bins"};
-  o2fw::Configurable<int> selPrim{"sel-prim", 1, "1 select primaries, 0 select all particles"};
-  o2fw::Configurable<int> makeEff{"make-eff", 0, "Flag to produce the efficiency with TEfficiency"};
-
-  o2fw::OutputObj<TList> list{"Efficiency"};
-  o2fw::HistogramRegistry histos{"Histos", {}, o2fw::OutputObjHandlingPolicy::AnalysisObject};
-
-  void init(o2fw::InitContext&)
+  Configurable<float> etaMin{"eta-min", -3.f, "Lower limit in eta"};
+  Configurable<float> etaMax{"eta-max", 3.f, "Upper limit in eta"};
+  Configurable<float> phiMin{"phi-min", 0.f, "Lower limit in phi"};
+  Configurable<float> phiMax{"phi-max", 2.f * M_PI, "Upper limit in phi"};
```

Comment:
The `M_PI` macro is not a part of the standard and its definition (or even existence) depends on the compiler.
A defined constant should be used instead, (there are `o2::constants::math::TwoPI` and `o2::constants::math::PI`.
@ktf I've seen `M_PI` used in many other places in O2. Should it not be discouraged in general?

Reply:
This is some heritage from the QA task, I removed it